portfolio/src/main/webapp/index.html

```diff
@@ -7,7 +7,7 @@
     <script src="script.js"></script>
```

Comment:
Can you please modify the PR description to reflect that you added the option to select a number of comments?

Reply:
What I added is the solution to the problem that the number of comments didn't stay selected after a comment submition (that is after page reload). I will modify the description of this PR and erase the last PR which was the one that actually added this.